content/docs/getting-started/installing.md

```diff
@@ -5,64 +5,85 @@ weight: 2
 
 ## From extension stores
 
-Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
+Scratch Addons is available in both Chrome Web Store and Firefox Add-ons.
 
-- Chrome Web Store (for Google Chrome, Opera, Brave, Vivaldi, Microsoft Edge (>79), and other Chromium-based browsers)  
+- Chrome Web Store (for Google Chrome, Opera, Brave, Vivaldi, Microsoft Edge Chromium (>79), and other Chromium-based browsers)  
   https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco  
 - Add-ons for Firefox (for Mozilla Firefox)  
   https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/  
 
-## From GitHub
-
-[The releases tab](https://github.com/ScratchAddons/ScratchAddons/releases) contains all builds of Scratch Addons, as well as the mirror of the store builds hosted on both servers.
```

Comment:
Why though? This is fine.

Reply:
@Hans5958 I suppose it is, but it's not a tab - it's a page.

Reply:
Fixed, I made a few changes though.
```markdown
### About GitHub releases

[The releases page](https://github.com/ScratchAddons/ScratchAddons/releases) contains the code and installation files for all development builds of Scratch Addons, as well as the mirror of the store builds.
```

Reply:
Technically it is a tab looking at how it is designed on the header on GitHub, but calling it a page is also fine.